<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resourse.qrc">:/images/register.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tb_page">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>371</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_login">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QLabel" name="lb_tel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>手机号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;合法手机号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
      </font>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;长度不超过15，只允许字母、数字、下划线的组合&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
      </font>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_register">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QLabel" name="lb_name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>昵称：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tel_register">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>手机号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password_register">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;长度不能超过20，不允许有特殊字符&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel_register">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;合法手机号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password_register">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;长度不超过15，只允许字母、数字、下划线的组合&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit_register">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>290</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
      </font>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear_register">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>290</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
      </font>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resourse.qrc"/>
 </resources>
 <connections/>
</ui>
